internal/clients/team4/speaker.go

```diff
@@ -86,22 +174,26 @@ func (s *speaker) DecideAnnouncement(ruleMatrix rules.RuleMatrix, result bool) s
 
 // CallJudgeElection is called by the legislature to decide on power-transfer
 // COMPULSORY: decide when to call an election following relevant rulesInPlay if you wish
-/*func (s *speaker) CallJudgeElection(monitoring shared.MonitorResult, turnsInPower int, allIslands []shared.ClientID) shared.ElectionSettings {
+func (s *speaker) CallJudgeElection(monitoring shared.MonitorResult, turnsInPower int, allIslands []shared.ClientID) shared.ElectionSettings {
 	// example implementation calls an election if monitoring was performed and the result was negative
 	// or if the number of turnsInPower exceeds 3
 	var electionsettings = shared.ElectionSettings{
 		VotingMethod:  shared.BordaCount,
 		IslandsToVote: allIslands,
 		HoldElection:  false,
 	}
+	//there are more important things to do...
+	if s.getSpeakerBudget() < s.getHigherPriorityActionsCost("AppointNextJudge") {
+		return electionsettings
+	}
 	if monitoring.Performed && !monitoring.Result {
 		electionsettings.HoldElection = true
 	}
 	if turnsInPower >= 2 {
```

Comment:
I know this isn't something you did, but it'd be nice to have it updated.

if (turnsInPower >= 2) || (monitoring.Performed && !monitoring.Result)